### ISSUE TITLE
Support auto-generated content for list endpoints

### ISSUE THREAD

**Issue by trusch** [caalberts/localroast#10]
It would be great to have auto-generated values for the responses especially for list endpoints. Mocking a large response list is otherwise just a very dumb task

**Comment by caalberts** [comment]
I agree with this. I have done this previously when mocking using Sinatra is to have dynamically generated values.

How do you see this feature being supported though?

I’m thinking something similar to https://github.com/stympy/faker, using keywords as values, like `{“id”: :uuid}`. It needs to be easily parsed though.
